docs/DeveloperGuide.md

```diff
@@ -604,42 +604,230 @@ testers are expected to do more *exploratory* testing.
+        "Charlotte Oliveiro","charlotte@example.com","@charlotte","[neighbours]","Charlotte","",""
+        "David Li","lidavid@example.com","@david","[family]","david","",""
+        "Irfan Ibrahim","irfan@example.com","@irfan","[classmates]","Irfan","",""
+        "Roy Balakrishnan","royb@example.com","@roy","[colleagues]","Roy","",""
+       ```
+   2. Test case: `import path/data/example.csv`
+      Expected: `example.csv` is imported. 
+   
+   3. Test case: `import path/data/x.csv`
+      Expected: No data is imported. Details of the error is shown in status message.
+   
+   4. Other incorrect import commands to try: `import`, `import path/`<br>
+      Expected: Similar to previous.
 
-1. Dealing with missing/corrupted data files
+### Export CSV
+1. Export list of persons to a CSV file.
+    1. Test case: `export path/data/example.csv`
+       Expected: `example.csv` is exported. Details shown in status message.
+   
+    2. Test case: `export`
+       Expected: No data is exported. Details of the error is shown in status message.
+   
+    3. Other incorrect export commands to try: `export path/`<br>
+       Expected: Similar to previous.
+
+### Filtering for list of persons
+1. Filter for persons based on tag
+   1. Test case: `filter t/friends t/colleague`.
+      Expected: List shows contacts with tags `friends` or `colleague`.
+   
+   2. Test case: `filter`
+      Expected: No change in list. Details of error shown in status message.
+   
+   3. Other incorrect filter commands to try: `filter s/`, `filter t/`<br>
+      Expected: Similar to previous.
+
+### Sorting list of persons
+1. Sort contacts based on field(`github, name, telegram or reset`) of given person
+   1. Test case: `sort name order/asc`
+      Expected: List sorted based on name in ascending order. Details shown in status message. 
+   
+   2. Test case: `sort email order/asc`
+      Expected: List is not sorted. Details of error shown in status message. 
+   
+   3. Other incorrect sort commands to try: `sort order/asc`, `sort order/desc`
+      Expected: Similar to previous.
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+### Finding persons
+1. Finding persons based on name.
+   1. Prerequisite: `John Doe`, `Alice`, and `Cindy` exists in KonTActs.
+   
+   2. Test case: `find john doe alice`
+      Expected: `John Doe` and `Alice` is found and revealed in the list. Details shown in status message.
+   
+   3. Test case: `find`
+      Expected: No change. Details of error shown in status message.
+   
+### Viewing a person
+1. Viewing a detailed info person
+    1. Test case: `view n/John Doe`
+       Expected: View window for `John Doe` appears. Details shown in status message.
+   
+    2. Test case: `view n/`
+       Expected: No view window appears. Details of error shown in status message.
+   
+    3. Other incorrect view commands to try: `view`, `view n`
```

Comment:
just typing `view` is not an incorrect command since it closes the viewWindow. Maybe we can remove `view` from the list of incorrect view commands?